README.md

```diff
@@ -1,6 +1,4 @@
-# kubernetes-nmstate
-
-<img src="logo/fullcolor.png" alias="project logo" />
+<img src="docs/assets/images/logo/fullcolor.png" alias="project logo" />
```

Comment:
Logos should stay under `logo/`. They are assets of the project, not of the documentation. Do not hide them down there.

Reply:
I will create a link from docs to logo doing the opposite is not working and I prefer not to duplicate stuff.

Reply:
Symlinks not working with markdown